### PR TITLE
Update Troubleshooting Docs

### PR DESCRIPTION
**Pull request by rcoletti116**
Instructions for finding IP address in iRobot app.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,9 +100,14 @@ Click on any of the items below to expand the corresponding answer.
 <details>
   <summary>I don't know what my Roomba's IP address is</summary>  
 
-  >In order to find your Roomba's IP address, you'll need to open up your Router Admin Panel and look for a list of devices. Once you identify the Roomba, you should see an associated IP address, however, this process will be different for each type of router.
+  >You can find your Roomba's IP Address in the iRobot app. Open the app and choose your Robot. Scroll down to the botom and find Robot Settings. Click Wi-Fi Settings and then Robot Wi-FI Details. You will find your IP address and various other network goodies here.
+  
+  ![Alt Text](https://github.com/rcoletti116/homebridge-roomba2/blob/rcoletti116-docsfiles/trim.1BD89A46-80F9-4FCB-A04B-4A610D403D4F.gif)
+
+
+  >You can find your Roomba's IP Address in the iRobot app. Open the app and choose your Robot. Scroll down to the botom and find Robot Settings. Click Wi-Fi Settings and then Robot Wi-FI Details. You will find your IP address and various other network goodies here.
```
Comment (karlvr) [inline README.md:108]: This paragraph is in here twice.

File: README.md [hunk 2]
```diff
@@ -100,9 +100,14 @@ Click on any of the items below to expand the corresponding answer.
 <details>
   <summary>I don't know what my Roomba's IP address is</summary>  
 
-  >In order to find your Roomba's IP address, you'll need to open up your Router Admin Panel and look for a list of devices. Once you identify the Roomba, you should see an associated IP address, however, this process will be different for each type of router.
+  >You can find your Roomba's IP Address in the iRobot app. Open the app and choose your Robot. Scroll down to the botom and find Robot Settings. Click Wi-Fi Settings and then Robot Wi-FI Details. You will find your IP address and various other network goodies here.
```
Comment (karlvr) [inline README.md:103]: Bottom is misspelled

File: README.md [hunk 3]
```diff
@@ -100,9 +100,14 @@ Click on any of the items below to expand the corresponding answer.
 <details>
   <summary>I don't know what my Roomba's IP address is</summary>  
 
-  >In order to find your Roomba's IP address, you'll need to open up your Router Admin Panel and look for a list of devices. Once you identify the Roomba, you should see an associated IP address, however, this process will be different for each type of router.
```
Comment (karlvr) [inline README.md:103]: I'm not sure we should remove the previous instructions. I think that can be an Alternatively… paragraph?

File: README.md [hunk 4]
```diff
@@ -100,9 +100,14 @@ Click on any of the items below to expand the corresponding answer.
 <details>
   <summary>I don't know what my Roomba's IP address is</summary>
```
Comment (karlvr) [inline README.md:101]: How about we change this to _Finding my Roomba's IP address_

**Review by karlvr — CHANGES_REQUESTED** [review]

**Review by karlvr — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -98,11 +98,15 @@ as to provide HomeKit with Roomba's status only when it requests it.
 Click on any of the items below to expand the corresponding answer.
 
 <details>
-  <summary>I don't know what my Roomba's IP address is</summary>  
+  <summary>Finding my Roomba's IP sddress</summary>
```
[the remark this answers is not in the document]
Comment (rcoletti116) [reply README.md:101]: Ugh, sorry! I'll fix it shortly.

File: config.schema.json [hunk 6]
```diff
@@ -55,7 +55,7 @@
                 "title": "Show docking status as a contact sensor",
                 "required": false
             },
-            "noDockOnStop": {
+             "noDockOnStop": {
```
Comment (karlvr) [inline config.schema.json:58]: I don't think this change is supposed to be in this PR!

**Review by karlvr — CHANGES_REQUESTED** [review]
Just that one glitch with the unexpected change.

**Review by karlvr — APPROVED** [review]

**Comment by karlvr** [comment]
@rcoletti116 thank you!

**Pull request merged**